scalableminds/webknossos
# Prevent accidental dismissal of team role modal view

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Register a new user. Log in with an administrator account and go to the users page. Activate the user, it should not be possible to dismiss the team role selection modal that's popping up by clicking somewhere else than the two buttons.

### Issues:
- fixes #3361 

------
